### DISCUSSION TITLE
Select default theme

### DISCUSSION THREAD

**Discussion by Syndlex** [plantuml/plantuml#1343] [Ideas]
Hey, is it possible to change the default theme?

It would be really nice if I could change the theme with an environment variable. 
We are using the adoc Integration and it would be nice to override the default theme without setting the theme on every new plantuml diagram.

Edit:
I found the code where i could implement this. 
If this is a desired change I would create a PR

**Comment by arnaudroques** [comment]
That sound interesting!
Before providing some code, could you describe us more precisely what you have in mind:

- A new directive?
- Some flag in the command line?
- Some environment variable?

Thanks!
